torch/_tensor_docs.py

```diff
@@ -1717,10 +1717,11 @@ def callable(a, b) -> number
 expand(*sizes) -> Tensor
 
 Returns a new view of the tensor with singleton dimensions expanded
-to a larger size.
+to a larger size. -1 means not changing the size on that dimension.
```

Comment:
nit: it's ambiguous what -1 means here...the size of a single dimension or the value of *sizes.  How about something like:

"Passing -1 as the size for a dimension means not changing the size of that dimension.  For example, if y is a tensor of size (1,10) then calling y.expand(3,-1) will expand y to be of size (3,10).